drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/data_warehouse_upload_job.rb

```diff
@@ -85,11 +87,12 @@ def hmis_csv_export
       export = HmisExternalApis::AcHmis::Exporters::HmisExportFetcher.new
       export.run!
 
-      hash = Digest::MD5.hexdigest(export.content)
+      content = export.hmis_zip.download
```

Comment:
Thanks @eanders! This seems ok. It might be slightly cleaner to adjust the `content` accessor on `HmisExternalApis::AcHmis::Exporters::HmisExportFetcher` instead. But its fine with me either way, this code is not touched frequently.

In `drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/hmis_export_fetcher.rb`:
```diff
-    delegate :content, to: :export
+    def content
+      export.hmis_zip.download
+    end
```

Reply:
I pushed this since it also fixes the test mocks that were failing. I ran it locally to confirm as well!

Reply:
Thanks!